### PR TITLE
Update francesinha.md

### PR DESCRIPTION
**Pull request by lisbonjoker**
> Did you mean to have `sausage` and `fresh sausage` separately?

It was supposed to be linguiça (type of Portuguese sausage), so people understand what was it.
I forgot to change it in the end. 

_Originally posted by @hallzy in https://github.com/LukeSmithxyz/based.cooking/pull/286#discussion_r598202331_

```suggestion
- linguiça (type of Portuguese sausage)
```
> Maybe it's just me, but I had no idea what `linguiça` was, so having just a short description might be helpful to others.

Added to the description of the recipe.

_Originally posted by @hallzy in https://github.com/LukeSmithxyz/based.cooking/pull/286#discussion_r598202575_

### CODE REVIEW

File: src/francesinha.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Francesinha
 
-Francesinha is a traditional sandwich from the city of Porto. The francesinha in its special sandwich variation consists of sausage, fresh sausage, ham, cold meat and beef steak, covered with melted cheese.
+Francesinha is a traditional sandwich from the city of Porto. The francesinha in its special sandwich variation consists of linguiça (type of Portuguese sausage), fresh sausage, ham, cold meat and beef steak, covered with melted cheese.
```
Comment (zyansheep) [inline src/francesinha.md:3]: There are 2 types of sausage?

Comment (lisbonjoker) [reply src/francesinha.md:3]: https://en.wikipedia.org/wiki/Lingui%C3%A7a

Comment (lisbonjoker) [reply src/francesinha.md:3]: Might make an article for it as well.

**Comment by LukeSmithxyz** [comment]
Remove the double mention of the sausage.

File: src/francesinha.md (hunk 1 above)
Comment (DGKSK8LIFE) [reply src/francesinha.md:3]: chad

**Comment by lisbonjoker** [comment]
> Remove the double mention of the sausage.

Will do

**Pull request merged**